aws/src/main/java/org/apache/iceberg/aws/s3/S3OutputFile.java

```diff
@@ -62,6 +63,9 @@ public PositionOutputStream createOrOverwrite() {
       return new S3OutputStream(client(), uri(), awsProperties());
     } catch (IOException e) {
       throw new UncheckedIOException("Failed to create output stream for location: " + uri(), e);
+    } catch (SecurityException e) {
```

Comment:
as discussed in the block below, we can remove this catch. AccessControlException also hides the original exception, so better to not use it.

Reply:
Removed the special handling for `SecurityException`.